db/models/interview.js

```diff
@@ -139,7 +139,114 @@ InterviewerCalendlyLinks.init(
   { sequelize, modelName: "InterviewerCalendlyLinks" }
 );
 
+class InterviewQuestions extends Model {}
 
+InterviewQuestions.init(
```

Comment:
For future reference
Model names should be singular words, not plural.
This way, when we say
`InterviewQuestion.create()` it makes sense, but
`InterviewQuestions.create()` is misleading because it doesn't create multiple questions.